lib/cli/kit/ini.rb

```diff
@@ -15,8 +15,12 @@ module Kit
     class Ini
       attr_accessor :ini
 
-      def initialize(path = nil, default_section: nil, convert_types: true)
-        @config = File.readlines(path) if path && File.exist?(path)
+      def initialize(path: nil, config: nil, default_section: nil, convert_types: true)
```

Comment:
Isn't it that for certain ruby versions, changing an argument from positional to keyword can cause BC breaks?

Reply:
We haven't tried insanely hard to avoid backwards compatibility issues when they're relatively obvious like this one, but at the same time, I'd prefer to avoid them unless they're worth inflicting some headache, which IMO this one isn't. I'd rather see path continue to be positional and just add `config` as another kwarg.